src/cloudai/workloads/ai_dynamo/ai_dynamo.sh

```
@@ -641,15 +641,25 @@ function launch_decode()
+    # - KV event port: one per worker
+    local nixl_port=$((base_nixl_port + (i * tp_size)))
+    local kv_event_port=$((base_kv_event_port + i))
 
-    log "Launching decode worker $i on GPUs $gpu_list"
+    log "Launching decode worker $i on GPUs $gpu_list (NIXL port: $nixl_port, KV event port: $kv_event_port)"
     log "Decode cmd: ${dynamo_args["decode-cmd"]} $(array_to_args decode_args) ${decode_args["--extra-args"]}"
     CUDA_VISIBLE_DEVICES=$gpu_list \
+      VLLM_NIXL_SIDE_CHANNEL_PORT=$nixl_port \
+      DYN_VLLM_KV_EVENT_PORT=$kv_event_port \
       ${dynamo_args["decode-cmd"]} \
       $(array_to_args decode_args) ${decode_args["--extra-args"]} > $log_file 2>&1 &
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)